nifi-nar-bundles/nifi-enrich-bundle/nifi-enrich-processors/src/main/java/org/apache/nifi/processors/AbstractEnrichIP.java

```diff
@@ -80,6 +90,10 @@ public abstract class AbstractEnrichIP extends AbstractProcessor {
             .description("Where to route flow files after unsuccessfully enriching attributes because no data was found")
             .build();
 
+    enum MessageLogLevel {
+        debug, info, warn, error
+    }
```

Comment:
Is there a reason for using lowercase values? Recommend using the conventional uppercase:
```suggestion
    enum MessageLogLevel {
        DEBUG, INFO, WARN, ERROR
    }
```